### DISCUSSION TITLE
how to cancel pending inscriptions?

### DISCUSSION THREAD

**Discussion by punk3700** [ordinals/ord#1596] [General]
i have a bunch of pending inscriptions that have been sitting in the mempool in the last few days... is there any way to cancel them?  thanks!

**Comment by casey** [comment]
Are they being sent, or being inscribed?

The wallet opts-all transactions into RBF, which will allow them to be fee bumped. However, the wallet doesn't have this functionality yet. I just opened an issue to start looking in to this: https://github.com/casey/ord/issues/1519
